src/view/styles.ts

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+export const PageWrapper = styled.div`
+  height: 90vh;
```

Comment:
[fix] You don't need this, so you can just remove this line.

Reply:
You can get rid of all styles of PageWrapper and just add a `padding-top` to add some space between the top and the component.